lib/bolt/cli.rb

```diff
@@ -114,12 +114,8 @@ def parse
       @config = if options[:configfile]
                   Bolt::Config.from_file(options[:configfile], options)
                 else
-                  project = if options[:boltdir]
-                              Bolt::Project.create_project(options[:boltdir])
-                            else
-                              Bolt::Project.find_boltdir(Dir.pwd)
-                            end
-                  Bolt::Config.from_project(project, options)
+                  boltdirpath = options[:boltdir] || Dir.pwd
+                  Bolt::Config.from_project(Bolt::Project.find_boltdir(File.expand_path(boltdirpath)), options)
```

Comment:
I think this will cause the user-level project to be used if you pass `--project` with a directory that doesn't exist / doesn't have a `bolt.yaml`/`bolt-project.yaml`, which is undesirable.

Reply:
That's a good point - I updated to first look for a Boltdir, otherwise use the specified path.